src/test/unit/format-numbers.test.js

```diff
@@ -0,0 +1,13 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+// @flow
+
+import { formatNumber } from 'firefox-profiler/utils/format-numbers.js';
+
+describe('formatNumber', () => {
+  it('return 0 without digits when called with 0', () => {
+    expect(formatNumber(0)).toBe('0');
```

Comment:
I wonder if we should also explicitly test formatNumber with values provided for the optional parameters. I know their default values are non-0, but the default values could be changed in the future.

Reply:
Maybe, but I didn't want to do all tests for all cases here ;-) but now there's a canvas to add more tests when needed.